package.json

```diff
@@ -112,7 +111,7 @@
     "lint:php": "composer lint",
     "lint:php:fix": "composer lint-fix",
     "packages-update": "wp-scripts packages-update",
-    "precommit": "./bin/pre-commit.sh",
+    "prepare": "git config --local core.hooksPath .githooks",
```

Comment:
This won't always work, but I like it more than importing [husky](https://www.npmjs.com/package/husky) and updating its versions.